packages/rrweb-snapshot/src/snapshot.ts

```diff
@@ -1008,7 +1008,10 @@ export function serializeNodeWithId(
     id = genId();
   }
 
-  const serializedNode = Object.assign(_serializedNode, { id });
+  const serializedNode: serializedNode & {
+    id: number;
+    chromaticAdoptedStylesheets?: Array<string | null>;
```

Comment:
I don't think we would want to keep this chromatic-specific field if submitting a PR to rrweb, but it's good enough for now